crates/net/eth-wire/src/types/blocks.rs

```diff
@@ -68,11 +68,17 @@ impl From<Vec<H256>> for GetBlockBodies {
 
 /// The response to [`GetBlockBodies`], containing the block bodies that the peer knows about if
 /// any were found.
-#[derive_arbitrary(rlp, 1)]
+#[derive_arbitrary(rlp, 16)]
```

Comment:
what does the number represent in this derive macro?

Reply:
the number of test cases, default is 256, but we set it to 1 previously because it was expensive. with these changes we can bump it to 16, and while still relatively costly, this is manageable. imo better to have more test cases than a single large test case